### PR TITLE
feat(HOME-3407-docs): documentation for v10 performance

### PR DESCRIPTION
**Pull request by DatChive**

### CODE REVIEW

**Comment by unjar** [comment]
👍

File: api/v10/index.raml [hunk 1]
```diff
@@ -183,3 +183,26 @@
...
+        get:
+          description: |
+            ### Get Account Performance
+          queryParameters:
+            dateAfter:
+              type: date
+              required: true
+            dateBefore:
+              type: date
+              required: true
```
Comment (yuan616510) [inline api/v10/index.raml]: Is it required?

Comment (yuan616510) [reply api/v10/index.raml]: Requiredness for  ```dateAfter``` ```dateBefore``` on V10 controllers should be consistent. Can @kvhswong  and @bondricek confirm the rule?

Comment (DatChive) [reply api/v10/index.raml]: yes

**Comment by DatChive** [comment]
Comments addressed

File: api/v10/index.raml [hunk 2]
```diff
@@ -183,3 +183,25 @@
...
+            200:
+              body:
+                application/json:
+                  schema: !include schema/getAccountPerformance.res.json
+                  example: !include examples/getAccountPerformance.res.json
+            400:
+              description: |
+                Can be one of the following reasons:
+
+                Invalid firmGuid, Invalid accountGuid, firm not associated with account, beforeDate is before afterDate, missing beforeDate
```
Comment (bondricek) [inline api/v10/index.raml]: can remove` firm not associated with account`

**Pull request merged**